src/components/Settings/AccountTab.react.js

```diff
@@ -454,6 +520,7 @@ function mapStateToProps(store) {
     email: store.app.email,
     accessToken: store.app.accessToken,
     avatarType: store.settings.avatarType,
+    userSettingsViewedByAdmin: store.settings.userSettingsViewedByAdmin,
```

Comment:
Instead of checking if email !== null everywhere, check it here and sent appropriate store values via props.